spec/services/spree_shopify_importer/data_parsers/stock_items/base_data_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.describe SpreeShopifyImporter::DataParsers::StockItems::BaseData, type: :service do
+  subject { described_class.new(spree_stock_location, shopify_inventory_level) }
+
+  let!(:shopify_inventory_level) { build_stubbed(:shopify_inventory_level, inventory_item_id: shopify_variant.inventory_item_id) }
+  let!(:shopify_variant) { create(:shopify_variant) }
```

Comment:
it's still better to stub searching object instead of create something in a DB, because in this test you shouldn't check that rails find works properly, you should just check your custom logic, so, you can just stub all finds

Reply:
👍